### DISCUSSION TITLE
Version

### DISCUSSION THREAD

**Discussion by GBInformatics** [SpotX-Official/SpotX-Bash#18] [Q&A] [answered]
Bonjour,
J'avais une version de spotify, que j'ai supprimé par erreur qui se nommait spotifyglitchh. Est-ce celle-ci en modifier ?

**Comment by jetfir3** [comment]
I tried translating your message and I still am not understanding what you are asking. 

What is "spotifyglitchh"? 
What does that have to do with SpotX-Bash?
SpotX-Bash is to be used with official Spotify desktop client for macOS and Linux to block ads and enable experimental features.
The README and FAQ should explain what this does.

**Comment by GBInformatics** [comment]
I was going to install a version that would block them. The icon on my desktop was spotifyglitchh. I was just asking if it was your version that was named like that or not because I can't find it.

**Comment by jetfir3** [comment, marked as answer]
No, unfortunately I do not know what Spotifyglitchh is. 

This project has always been based on SpotX naming.

Searching on Google  for the name you mentioned leads to this repo - https://github.com/theglitchh/spotify-adblock-linux
which seems to just be a script to install an older spotify deb file and then use the spotify adblock from https://github.com/abba23/spotify-adblock

SpotX blocks ads differently than that repo and also removes things like the empty ad banners and more.

**Reply by TheSerphh** [reply]
well hello hello

**Comment by GBInformatics** [comment]
I'll continue with you if it's functional

**Comment by jetfir3** [comment]
You will first need to install Spotify client. then patch with SpotX-Bash.
